ci/build_linux.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+set -e -u -x -o pipefail
+
+cd "$(dirname "$0")/../"
+
+source ci/profiling.sh
+
+markStartOfBlock "$0"
+
+source ci/prepare_build_environment.sh
+
+markStartOfBlock "Build the StarterProject (Linux)"
+  # Build each target to ensure scripts are correct, skipping code generation on all but the first to save some time.
+  Game/Scripts/BuildWorker.bat "StarterProjectEditor" "Win64" "Development" "StarterProject.uproject"
```

Comment:
Why do we need to build Win64 editor both here and in `build_windows.sh`?

Reply:
You need to have the editor built to be able to cross compile for linux

Reply:
At least that was the case in the past, not sure if we have changed anything that would have caused it to change, but I think this is still the case.